presto-docs/src/main/sphinx/sql/source.rst

```diff
@@ -0,0 +1,23 @@
+======
+SOURCE
+======
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SOURCE file-path
+
+Description
+-----------
+
+Execute all complete sql statements in the given file.
```

Comment:
We should document this as part of the CLI docs. I actually don't think `USE` should be documented as a SQL statement, either, since it is only in the CLI (and also shouldn't be part of the main SQL grammar).